__docs__/wonder-blocks-cell/detail-cell.stories.tsx

```diff
@@ -123,8 +123,9 @@ DetailCellDisabled.parameters = {
 export const DetailCellWithCustomStyles: StoryComponentType = () => (
     <DetailCell
         title="Title for article item"
-        subtitle1="Subtitle for article item"
-        subtitle2="Subtitle for article item"
+        contentStyle={{
+            alignSelf: "flex-start",
+        }}
```

Comment:
**suggestion:** It would be nice adding a short description that this prop can be used in `docs.description.story`.